source/guide/3.2/plugin-softlayer.md

```diff
@@ -13,6 +13,20 @@ This section describes how to use a SoftLayer based cloud infrastructure in your
 For more information about SoftLayer, please refer to: [http://www.softlayer.com/](http://www.softlayer.com/).
 {%endsummary%}
 
+# Requirements
+  * A SoftLayer account, see [SoftLayer Authentication](#softlayer-authentication).
+  * [The commercial version of Cloudify](/goPro.html) 
+
+# Compatibility
+  The SoftLayer plugin uses the [SoftLayer for python 3.3.0](https://pypi.python.org/pypi/SoftLayer/3.3.0).
```

Comment:
The name is confusing, cause the 3.3.0 refers to the package while it might sound like it's for python 3.3. maybe we should remove the 3.3.0 and just link to, or write "The softlayer plugin uses the [Softlayer for python](link...) package, v. 3.3.0.